samples/DispatcherSample/Startup.cs

```diff
@@ -16,9 +19,61 @@ public void ConfigureServices(IServiceCollection services)
 
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+            app.Use(async (context, next) =>
+            {
+                await context.Response.WriteAsync("<p>Middleware 1</p>");
+                await next.Invoke();
```

Comment:
How does this work? Doesn't this need a parameter? I'd expect to see `next(context)`